inthe_am/taskmanager/models/taskstore.py

```diff
@@ -360,6 +360,8 @@ def delete(self, *args, **kwargs):
             if raise_for_failure:
                 raise
 
+        self.user.delete()
```

Comment:
For the record, I was intentionally leaving this user account in place (to later be automatically cleaned up by https://github.com/coddingtonbear/inthe.am/blob/db9d77fcc4ffc1bc37eab4789344e2fe414671f5/inthe_am/taskmanager/management/commands/taskstore.py#L189) so malicious or abusive accounts could potentially be identified.  There's basically no long-term logging for inthe.am for a variety of reasons; so if a user were to somehow use inthe.am in a malicious way but delete their account afterward, it'll make it quite a lot harder (if possible at all) to identify them so they can be blocked.  Realistically, that happens so very rarely that I suppose I'm not that worried about it, though.

Reply:
OK, understood! It makes sense.  I am rolling back it.